aws/fedora-coreos/kubernetes/workers/fcc/worker.yaml

```diff
@@ -130,6 +132,18 @@ storage:
           DefaultCPUAccounting=yes
           DefaultMemoryAccounting=yes
           DefaultBlockIOAccounting=yes
+    - path: /etc/containerd/config.toml
+      overwrite: true
+      contents:
+        inline: |
+          [plugins]
+            [plugins."io.containerd.grpc.v1.cri"]
+              enable_selinux = true
+              [plugins."io.containerd.grpc.v1.cri".cni]
+                bin_dir = "/opt/cni/bin"
+                conf_dir = "/etc/kubernetes/cni/net.d"
+                max_conf_num = 1
```

Comment:
you have to enable systemd to avoid running cgroupfs with containerd. Consider adding:
```
          [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
            systemdCgroup = true
```